shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -395,6 +397,7 @@ - (void)surfaceUpdated:(BOOL)appeared {
     [_engine.get() platformViewsController] -> SetFlutterViewController(self);
```

Comment:
I don't know this code, but would it make more sense to set `isRenderingFrames` to YES in the if of this statement to make it symmetrical?  It seems like an unexpected side-effect for `(void)callViewRenderedCallback` to set it.

Reply:
We present the surface before we actually are rendering frames

Reply:
I have to double check this - I'm realizing this might not get invoked again in a serize of push/pop/push

Reply:
No, it's ok - we install the first frame callback every time we appear.